contracts/carbon/CarbonController.sol

```diff
@@ -133,6 +141,22 @@ contract CarbonController is
         _setTradingFeePPM(newTradingFeePPM);
     }
 
+    /**
+     * @dev sets the custom trading fee for a given pair (in units of PPM)
+     *
+     * requirements:
+     *
+     * - the caller must be the admin of the contract
+     */
+    function setCustomTradingFeePPM(
+        Token token0,
+        Token token1,
+        uint32 newCustomTradingFeePPM
```

Comment:
I think that elsewhere in the code, we're using `Token[2] memory tokens` as function input.

Reply:
We use both :)
Reference: check out `createStrategy` and `_createStrategy`.
The public function uses `token0` and `token1`.
Anyway, this is not a big issue imo.